deepmd_utils/model_format/network.py

```diff
@@ -329,3 +329,62 @@ def call(self, x: np.ndarray) -> np.ndarray:
         for layer in self.layers:
             x = layer.call(x)
         return x
+
+
+class EmbeddingNet(NativeNet):
+    def __init__(
+        self,
+        in_dim,
+        neuron: List[int] = [24, 48, 96],
```

Comment:
I think the first layer is 1? Should it be written explicitly?

Reply:
Not really, we may pass the concatenated the s(r) and type through the embedding net.  see eq 5 of https://arxiv.org/pdf/2208.08236.pdf